test/models/post_test.rb

```diff
@@ -1,7 +1,18 @@
 require 'test_helper'
 
 class PostTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test 'presence of post title and description' do
+    @post = FactoryGirl.create(:post)
+    assert_equal @post.valid?, true, 'valid post'
```

Comment:
@anijat-cuelogic : can you tell me the use of 4th parameter in `assert_equal` method ?